### ISSUE TITLE
Add recurring bank transfers

### ISSUE THREAD

**Issue by codebeaulieu** [alpacahq/Alpaca-API#32]
I would like to set up a recurring (daily, weekly, biweekly, monthly) bank transfer through the portal.

**Comment by JonFurmanski** [comment]
+1 for this feature

**Comment by yunluyl** [comment]
+1

**Comment by cashjee** [comment]
+1

**Comment by sanjayj411** [comment]
Yes I would like to have recurring deposit either online or through api

**Comment by meetnaren** [comment]
+1

**Comment by codebeaulieu** [comment]
This really seems like an easy win. I would like to set up an account that I can passively invest a portion of my paychecks into. Without needing to manually create the transfer.

**Comment by bheem-palepuin** [comment]
+1 
Have been waiting for this feature for a while. It is painful to remember and transfer weekly/bi-weekly

**Comment by ghost** [comment]
+1

**Comment by gthom9** [comment]
+1

**Comment by Furb13** [comment]
Amazing this still hasn't been implemented over 4 years later. Honestly pathetic and no response from the developers.

**Comment by codebeaulieu** [comment]
> Amazing this still hasn't been implemented over 4 years later. Honestly pathetic and no response from the developers.

It's kinda insane. I just wrote a recurring deposit system at work and it took about 2 weeks to get it from start to finish. I understand if it's not a priority for the company, but at least say as much, rather than ignoring the request.

**Comment by deepeecee** [comment]
What are the odds that we can get this? ❤  This would be huuuuge!!
